lib/KImg.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+
+  <div>
+    <img
+      id="kimg"
```

Comment:
Using id could prove problematic especially when more than one `KImg` is used on a single page. It is on this basis that css class referencing is recommended instead